src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color1 = colorSupplier.getRandomColor().toString();
+        int number1 = getRandomNumber();
+        Ball ball = new Ball();
+        ball.setColor(color1);
+        ball.setNumber(number1);
+        return ball;
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/63095184/d0d543ab-6031-45d5-8375-e2227eaad8af)

Reply:
not fixed